### DISCUSSION TITLE
PopperJS website now redirects to floatingUI

### DISCUSSION THREAD

**Discussion by digitalrobin** [floating-ui/floating-ui#2817] [General]
Hi all! We have a legacy project, that depends heavily on the older popperJS. 

We use https://popper.js.org/ a _lot_ to debug issues, as our project is to big to migrate at the moment (and a lack of developers in the team that can spend time on it). 

However, https://popper.js.org/ keeps being re-routed to https://floating-ui.com/?utm_source=popper.js.org which is super annoying, as we were dependent on these docs.

Am i  doing something wrong? Or are other users also experiencing this?

**Comment by atomiks** [comment]
The homepage redirects, but not the actual docs, which you can read: https://popper.js.org/docs/v2/. This is also the first result on Google when searching, so it shouldn't be that disruptive. The Popper homepage doesn't really need to be seen anymore :)

**Reply by FezVrasta** [reply]
The docs link is also on the floating-ui readme
![CleanShot 2024-03-14 at 5  17 40@2x](https://github.com/floating-ui/floating-ui/assets/5382443/8459cd6f-ae0f-4d32-a8cd-0dcd869598d5)
